### PR TITLE
AP_TempertureSensor:clarify temp source for pitot tube

### PR DESCRIPTION
**Pull request by Hwurzburg**
NFC

### CODE REVIEW

**Comment by magicrub** [comment]
This is not necessarily correct. It could all happen within the autopilot. It's just a label.

**Comment by Hwurzburg** [comment]
> This is not necessarily correct. It could all happen within the autopilot. It's just a label.

please give an example....it always comes from an AP_Periph device which only talks over CAN to the autopilot

File: libraries/AP_TemperatureSensor/AP_TemperatureSensor_Params.cpp [hunk 1]
```diff
@@ -58,7 +58,7 @@ const AP_Param::GroupInfo AP_TemperatureSensor_Params::var_info[] = {
     // @Param: SRC
     // @DisplayName: Sensor Source
     // @Description: Sensor Source is used to designate which device's temperature report will be replaced by this temperature sensor's data. If 0 (None) then the data is only available via log. In the future a new Motor temperature report will be created for returning data directly.
-    // @Values: 0: None, 1:ESC, 2:Motor(not implemented yet), 3:Battery Index, 4:Battery ID/SerialNumber, 5: Pitot tube
+    // @Values: 0: None, 1:ESC, 2:Motor(not implemented yet), 3:Battery Index, 4:Battery ID/SerialNumber, 5: CAN based Pitot tube
```
Comment (tridge) [inline libraries/AP_TemperatureSensor/AP_TemperatureSensor_Params.cpp:61]: @magicrub says the pitot tube option isn't implemented yet, so lets just remove it from the list

**Review by tridge — CHANGES_REQUESTED** [review]

File: libraries/AP_TemperatureSensor/AP_TemperatureSensor_Params.cpp (hunk 1 above)
Comment (magicrub) [reply libraries/AP_TemperatureSensor/AP_TemperatureSensor_Params.cpp:61]: yup, lets remove "Pitot tube" enum for now until I get my AP_Heater library merged in which will use it. The enum is currently a placeholder but I agree that it's creating confusion right now.

Comment (Hwurzburg) [reply libraries/AP_TemperatureSensor/AP_TemperatureSensor_Params.cpp:61]: @tridge but this is only part of a larger code change...and the enum impacts this code, not just the switch statement.....its the reason I said this is CAN specific: https://github.com/ArduPilot/ardupilot/pull/24462
should that CAN change be reverted also?

**Review by magicrub — APPROVED** [review]

File: libraries/AP_TemperatureSensor/AP_TemperatureSensor_Params.cpp (hunk 1 above)
Comment (Hwurzburg) [reply libraries/AP_TemperatureSensor/AP_TemperatureSensor_Params.cpp:61]: I posted a PR that removes the unused ENUM as Tridge suggested...you guys decide

**Pull request merged**